aptos-move/aptos-native-interface/src/context.rs

```diff
@@ -118,4 +118,9 @@ impl<'a, 'b, 'c, 'd> SafeNativeContext<'a, 'b, 'c, 'd> {
     pub fn set_incremental_gas_charging(&mut self, enable: bool) {
         self.enable_incremental_gas_charging = enable;
     }
+
+    /// Returns true if the aggregator snapshots feature is enabled.
+    pub fn is_aggregator_snapshots_enabled(&self) -> bool {
```

Comment:
it's a preference thing, but I don't like is_ prefix when the rest of the name can function without, e.g. I think it applies in this case where aggregator_snapshots_enabled() already expresses that it's an indicator. But as you prefer.

Reply:
Done.